template.json

```diff
@@ -1,38 +1,42 @@
 {
   "package": {
     "dependencies": {
+      "@nimbl3/eslint-config-nimbl3": "2.1.1",
```

Comment:
Should we rebase to the latest `develop` branch as this package is the wrong one? [the fix](https://github.com/nimblehq/react-templates/pull/29) was already merged to `develop` branch 😄

Reply:
Yup! :) Done ;-)